### PR TITLE
Make php7.3 compatible

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

**Comment by rullzer** [comment]
same issue as in https://github.com/nextcloud/server/pull/11746

Will fix once that is in

**Comment by MorrisJobke** [comment]
> Will fix once that is in

It is merged.

**Comment by rullzer** [comment]
rebased

File: lib/versioncheck.php [hunk 1]
```diff
@@ -9,8 +9,8 @@
 	exit(-1);
 }
 
-// Show warning if > PHP 7.2 is used as Nextcloud is not compatible with > PHP 7.2 for now
-if (version_compare(PHP_VERSION, '7.3.0') !== -1) {
+// Show warning if > PHP 7.3 is used as Nextcloud is not compatible with > PHP 7.3 for now
+if (version_compare(PHP_VERSION, '7.4.0') !== -1) {
 	http_response_code(500);
 	echo 'This version of Nextcloud is not compatible with > PHP 7.2.<br/>';
```
Comment (MorrisJobke) [inline lib/versioncheck.php]: This also needs to be updated.

**Review by nickvergessen — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**